garak/generators/rest.py

```diff
@@ -193,6 +195,7 @@ def _call_model(
             data_kw: request_data,
             "headers": request_headers,
             "timeout": self.request_timeout,
+            "verify": self.verify_ssl,
```

Comment:
can we suppress the warning that this brings per requests?

```
home/lderczynski/anaconda3/envs/garak/lib/python3.12/site-packages/urllib3/connectionpool.py:1099: 
InsecureRequestWarning: Unverified HTTPS request is being made to host 'derczynski.com'. 
Adding certificate verification is strongly advised. 
See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
```

Could do something like
```
if self.verify_ssl is False:
  import urllib3
  urllib3.disable_warnings()
```
in the constructor

Reply:
Worth investigating, might make sense to use:
```
requests.packages.urllib3.disable_warnings()
```
I suspect it would have the same global effect on all `urllib3` warnings.

Looking into this for more limited suppression I noticed that `verify` can actually be a `bool` or `str` as you can pass the path to a CA used for verification instead of a boolean.  I think it may be valuable to expand to support that here as well.